src/lib/modernExtend.ts

```diff
@@ -1852,7 +1853,12 @@ export function electricityMeter(args?: ElectricityMeterArgs): ModernExtend {
                                 assertNumber(multiplier, property.multiplier);
                                 change = property.change * (divisor / multiplier);
                             }
-                            items.push({attribute: property.attribute, min: '10_SECONDS', max: 'MAX', change});
+                            let minval: ReportingConfigTime = '10_SECONDS';
+                            // In case min was provided, use that instead of default.
+                            if ('min' in property && 'forced' in property && property.forced && 'min' in property.forced && property.forced.min) {
+                                minval = property.forced.min ?? minval;
+                            }
+                            items.push({attribute: property.attribute, min: minval, max: 'MAX', change});
```

Comment:
```suggestion
                            items.push({attribute: property.attribute, min: property.forced?.min ?? '10_SECONDS' , max: 'MAX', change});
```

Does this work? (then the rest can be removed)

Reply:
The compiler also complains about the optional property chaining in the coalesce.
```
Property 'forced' does not exist on type '{ attribute: string; divisor: string; multiplier: string; forced: false | (MultiplierDivisor & { min?: ReportingConfigTime; }); change: number; } | ... 9 more ... | { ...; }'.
  Property 'forced' does not exist on type '{ attribute: string; change: number; }'.ts(2339)
```
 I tried many permutations but couldn't get it to accept anything in that single items.push statement. I even explored adding a placeholder forced property to power_factor being the only one without a forced but that caused other issues. Hence going back to the separate variable approach.

I'll update the PR with another commit bundling these changes.